.github/workflows/performance-testing.yml

```diff
@@ -2,55 +2,42 @@ name: performance testing
 
 on:
   pull_request:
-    branches: ['main']
+    branches:
+      - main
 
 jobs:
   performance:
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: ${{ vars.NODE_VERSION }}
+          node-version: '20.x'
```

Comment:
The node version should be configured through repository variable, please return back